tests/warehouses/spark_cfg.yml

```diff
@@ -0,0 +1,5 @@
+type: spark
```

Comment:
I put in some basics for now. We could add multiple `methods` and some other parameters to connect to different locations. See [`dbt-spark`](https://github.com/fishtown-analytics/dbt-spark#configuring-your-profile) as example.

Reply:
I think this is good start - we can add additional parameters to differentiate later